rosdep/base.yaml

```diff
@@ -2708,7 +2708,7 @@ pkg-config:
   debian: [pkg-config]
   fedora: [pkgconfig]
   freebsd: [pkg-config]
-  gentoo: [dev-util/pkgconfig]
+  gentoo: [dev-util/pkgconf]
```

Comment:
Both seems to exist:
* https://packages.gentoo.org/package/dev-util/pkgconfig
* https://packages.gentoo.org/package/dev-util/pkgconf

Why is the new one favored over the previous one?